buildpipeline/pipeline.json

```diff
@@ -18,7 +18,8 @@
         {
           "Name": "DotNet-CoreFx-Trusted-Linux",
           "Parameters": {
-            "PB_DockerTag": "debian82_prereqs_2"
+            "PB_DockerTag": "debian82_prereqs_2",
+            "PB_TargetQueue": "Debian.82.Amd64"
           },
           "ReportingParameters": {
             "OperatingSystem": "Debian 8.2",
```

Comment:
MC probably will want the "TargetQueue" build parameter specified for these.

```
"ReportingParameters": {
  "OperatingSystem": "Debian 8.2",
  "TargetQueue": "Debian.82.Amd64",
  ...
},
```

/cc @ChadNedzlek @Chrisboh